tests/server/configs/sjrk.storyTelling.server.testServerWithBaseTheme.json5

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "sjrk.storyTelling.server.testServerWithCustomTheme",
+    "options": {
+        "gradeNames": ["sjrk.storyTelling.server"],
+        "components": {
+            "server": {
+                "options": {
+                    "components": {
+                        "baseTheme": {
+                            options: {
+                                root: "./tests/testData/mockBaseTheme"
```

Comment:
Rather than directly overriding `root`, its calculation can be derived by defining configs via globalConfig and secureConfig, which will help to test sjrk.storyTelling.server.getThemePath() function too.

The same comment applies to definitions in tests/server/configs/sjrk.storyTelling.server.testServerWithCustomTheme.json5.